cli-commands/nodeos/subcommands/stop/messages.js

```diff
@@ -0,0 +1,21 @@
+const chalk = require('chalk');
+
+module.exports = {
+    'StoppingNodeos': () => { console.log(chalk.magentaBright('===== Stopping nodeos ... =====')); },
+    'NoRunningNodeos': () => { console.log(chalk.redBright('===== There is no running nodeos ... =====')); },
+    'SuccessfulProcessTermination': () => { console.log(chalk.blueBright('===== Instance terminated =====')); },
+    'UnsuccessfulProcessTermination': (error) => {
+        console.log(chalk.redBright('===== Unsuccessful instance termination =====')); 
+        console.log(error);
+    },
+    'SuccessfulCleanUp': () => { console.log(chalk.blueBright('===== Folder cleaned up =====')); },
+    'UnsuccessfulCleanUp': (error) => {
```

Comment:
Does not need this. Nodeos is started or stoped. Too much info maybe ?

Reply:
removed